### ISSUE TITLE
[feature request] Add telegram as an 'account' in SFOS so that we can use the share function

### ISSUE THREAD

**Issue by null-von-sushi** [blacksailer/depecher#118]
I am not sure if this is related, but it would help with consistency anyways. Please add telegram accounts to the 'accounts' section of Sailfish OS so that we can select it as a target when sharing. Often times I wanna share a screenshot with a group chat I am in, and the way I always used to do this is using the share function of the OS. 
 
However, while SFOS provides this, the only targets I have are bluetooth and email. Pressing add account lets me add VKontakte, Facebook, Google, Email etc. but depecher is not listed. It would be great if we could use the provided share button.

**Comment by raketti** [comment]
Could this help on the sharing issue: https://together.jolla.com/question/18536/is-there-any-social-sharing-api/?answer=87698#post-id-87698

GitLab repo: https://git.merproject.org/mer-core/transfer-engine

**Comment by blacksailer** [comment]
Yes, I amtrying to implement it, it is not such straight forward, require some time
